Sources/SourceKitBazelBSP/SharedUtils/Shell/CommandRunner.swift

```diff
@@ -20,11 +20,19 @@
 import Foundation
 
 protocol CommandRunner {
+    func execute(_ cmd: String, cwd: String?) throws -> Data
```

Comment:
Nice one! A small nit: I think we can actually keep the same name `run` name here, then each caller chooses if it wants the string or the raw data